client/extensions/woocommerce/components/share-widget/index.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+import { localize } from 'i18n-calypso';
+import React, { Component, PropTypes } from 'react';
+import url from 'url';
+
+/**
+ * Internal dependencies
+ */
+import SocialLogo from 'social-logos';
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/14339/commits/d53efa5922db1e46368e0085ee422ed80acfa7de